### PR TITLE
feat(app): Plate reader in LPC and CommandText, filter out plate reader lid as labware

### PR DESCRIPTION
**Pull request by smb2268**
fix PLAT-483, PLAT-525, PLAT-526, PLAT-547, RQA-3310

# Overview
This PR cleans up a few things for plate reader support in the UI, enables LPC, hides the labware lid from protocol setup and details, and wires up command text

## Test Plan and Hands on Testing
Reviewed the following locations to make sure plate reader lid does not display:
1. ODD protocol details labware list and map view
2. ODD protocol setup labware list and map view
3. Desktop protocol details labware list and map view
4. Desktop protocol setup labware list and map view

Looked at run preview for protocol that uses absorbance reader, view command text updates 
Screen Shot 2024-10-15 at 3 28 52 PM
Screen Shot 2024-10-15 at 3 28 37 PM

Ran LPC for protocol that uses absorbance reader

## Changelog
1. Add column 4 icon when there is an absorbance reader attached
6. Remove temporary skip of absorbance reader and lid in LPC, permanently skip all lids and add prep commands for absorbance reader
7. Filter out absorbance reader lid from display on ODD and desktop protocol details and setup including list view and map view for each
8. Add JS types for absorbance reader commands and wire them up to command text

## Review requests
Look over screenshots and command types

## Risk assessment
Low

### CODE REVIEW

**Review by ecormany — COMMENTED** [review]
Command text looks great, thank you!

File: app/src/assets/localization/en/protocol_command_text.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "absorbance_reader_open_lid": "Opening Absorbance Reader lid",
+  "absorbance_reader_close_lid": "Closing Absorbance Reader lid",
+  "absorbance_reader_initialize": "Initializing Absorbance Reader to perform {{mode}} measurement at {{wavelengths}}",
```
Comment (vegano1) [inline app/src/assets/localization/en/protocol_command_text.json:4]: @ecormany @smb2268 
Do we want to mention the `referenceWavelength`, which is optional and only available in `single` mode?

File: app/src/organisms/ModuleCard/AbsorbanceReaderData.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export const AbsorbanceReaderData = (
         data-testid="abs_module_data"
       >
         {t('abs_reader_lid_status', {
-          status: moduleData.lidStatus === 'on' ? 'open' : 'closed',
+          status: moduleData.lidStatus === 'on' ? 'closed' : 'open',
```
Comment (vegano1) [inline app/src/organisms/ModuleCard/AbsorbanceReaderData.tsx:49]: nice catch

**Review by vegano1 — APPROVED** [review]
This looks good, just left a small comment on the initialize command copy.

File: app/src/assets/localization/en/protocol_command_text.json (hunk 1 above)
Comment (ecormany) [reply app/src/assets/localization/en/protocol_command_text.json:4]: Hmm, can't hurt. We would tack on ` with reference of {{referenceWavelength}} nm`

Comment (smb2268) [reply app/src/assets/localization/en/protocol_command_text.json:4]: added!

**Pull request merged**